client/state/data-layer/wpcom/posts/revisions/index.js

```diff
@@ -87,9 +87,10 @@ export const receiveSuccess = ( { dispatch }, { siteId, postId }, next, revision
  * @param {Object} action Redux action
  */
 export const fetchPostRevisions = ( { dispatch }, action ) => {
-	const { siteId, postId } = action;
+	const { siteId, postId, postType } = action;
+	const resourceName = postType === 'page' ? 'pages' : 'posts';
```

Comment:
- associated action creator: nothing? `postType` is the most suited parameter to transmit that data in the action imho: using the `rest_base` in the action would leak a "data-layer" concept?

- data handler:
```
export const fetchPostRevisions = ( { dispatch, getState }, action ) => {
	const { siteId, postId } = action;
	const { siteId, postId, postType } = action;
	const resourceName = getPostType( getState(), siteId, postType ).rest_base ?
	(...)
} );
```